portfolio/src/main/webapp/script.js

```diff
@@ -89,3 +89,28 @@ window.onload = (event) => {
 		}
 	})
 }
+
+function drawChart(commentInfo, numberOfComments){
+  fetch(`/comment-${commentInfo}-data?numberOfComments=${String(numberOfComments)}`).then(response => response.json()).then((commentData) => {
+  trueCommentData = [];
```

Comment:
"Data" part in the variable name is kidna meaningless isn't it? Let's just call it trueComments